tools/Makefile.am.inc

```diff
@@ -16,21 +16,6 @@
 ## along with eos-metrics-instrumentation.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
-bin_PROGRAMS = tools/eos-network-id
-
-tools_eos_network_id_SOURCES = \
-	src/eins-network-id.h \
-	src/eins-network-id.c \
-	tools/eos-network-id.c \
-	$(NULL)
-
-tools_eos_network_id_CFLAGS = \
-	$(EOS_INSTRUMENTATION_CFLAGS) \
-	-I$(top_srcdir)/src \
-	$(NULL)
-
-tools_eos_network_id_LDADD = $(EOS_INSTRUMENTATION_LIBS)
-
 nodist_bin_SCRIPTS = tools/eos-label-location
```

Comment:
The problem appears to be that Automake is not making the `tools` subdirectory in the builddir. I had not tested an out-of-tree build.

My autotools-ese is too rusty to understand what can be done to cause automake to make the directory. I'll look more later.

Reply:
Ugh, maybe I should just chuck the meson port onto this branch and not think about this.

Reply:
Added a fix which is gross but does work.

Reply:
I think that's the right thing to do. You can the same in `data/Makefile.am.inc`.